src/Maybe.js

```diff
@@ -36,6 +36,18 @@ Maybe.isNothing = function(x) {
   return x === _nothing;
 };
 
+Maybe.toArray = function(m) {
```

Comment:
I thought about that, but wasn't sure where it should go.

Is it worth introducing a separate file that can hold functions that apply to `Foldable` instances? It seems a bit too fantasyland focused to end up in Ramda proper.